index.js

```diff
@@ -0,0 +1,151 @@
+const {Client} = require("pg");
+
+let client;
+
+    if (process.env.NODE_ENV === "test") {
+        client = new Client("postgres://localhost:5432/costume_shop_db_test");
+        client.on("error", (error) => {
+            console.error(error.stack())
+        })
+    } else {
+        client = new Client("postgres://localhost:5432/costume_shop_db_dev");
+        client.on("error", (error) => {
+            console.error(error.stack())
+        })
+    }  
+
+
+const createTables = async () => {
+    await client.query(`
+        DROP TABLE IF EXISTS costumes;    
+    `)
+    await client.query(` 
+        CREATE TABLE costumes(
+            id SERIAL PRIMARY KEY,
+            name VARCHAR(80) NOT NULL,
+            category VARCHAR NOT NULL CHECK(category IN(
+                'adult', 
+                'child', 
+                'baby', 
+                'pet'
+            )), 
+            gender VARCHAR NOT NULL CHECK(gender IN(
+                'male',
+                'female',
+                'unisex'
+            )),
+            size VARCHAR(10) NOT NULL,
+            type VARCHAR(80) NOT NULL,
+            stock_count INTEGER NOT NULL,
+            price FLOAT NOT NULL 
+        );
+    `)
+    // Selecting table created by me, which should just be 'costumes'
+    const table = await client.query(`
+        SELECT * FROM pg_catalog.pg_tables WHERE tableowner='darayazdani';
+    `)
+    console.log(table.rows[0])
+    // Returning the name of the table I created
+    return table.rows[0].tablename;
+}
+
+const createCostume = async (
+    costumeName, 
+    category, 
+    gender, 
+    size, 
+    type, 
+    stockCount, 
+    price
+    ) => {
```

Comment:
```suggestion
const createCostume = async ({
    costumeName, 
    category, 
    gender, 
    size, 
    type, 
    stockCount, 
    price
    }) => {
```

Nit: this function takes enough arguments that I would recommend using a destructured args object instead of just receiving each individual argument positionally. This would make it easy to then add a typescript type for the args object (here is one breakdown of this approach: https://mariusschulz.com/blog/typing-destructured-object-parameters-in-typescript)

This has the benefit that it protects you from ever accidentally putting the args in the wrong order and ending up with the values mixed up - esp. when some args share a typescript type.

Reply:
Excellent! Good idea.